coremltools/converters/mil/frontend/torch/ops.py

```diff
@@ -4353,9 +4353,31 @@ def logical_xor(context, node):
     y = mb.cast(x=y, dtype="bool")
     context.add(mb.logical_xor(x=x, y=y, name=node.name))
 
+
+def _nonzero_as_tuple(context, node, x):
+    non_zero = mb.non_zero(x=x)
+
+    result = []
+    for i in range(x.rank):
+        result.append(mb.slice_by_index(x=non_zero,
+                                        begin=[0, i],
+                                        end=[-1, -1], # Will be ignored, but is required
+                                        end_mask=[True, True],
```

Comment:
It is a little but confusing to set the `end_mask[1] = True`, since the `squeeze_mask[1]` takes over.
I would leave the value as default, which is `False`.
i.e., `end_mask=[True, False]`

Reply:
Ok. Done.